### PR TITLE
publish 0.8.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by cho-m** [comment]
XCode requirement is probably 12.5 for Swift 5.4.
Though, that won't help Catalina node.
Probably still need https://github.com/Homebrew/homebrew-core/pull/76206#issuecomment-828879671

File: Formula/publish.rb [hunk 1]
```diff
@@ -13,7 +13,8 @@ class Publish < Formula
   end
 
   # https://github.com/JohnSundell/Publish#system-requirements
-  depends_on xcode: ["11.4", :build]
+  depends_on xcode: ["12.4", :build]
+  depends_on macos: :catalina
```
Comment (carlocab) [inline Formula/publish.rb:17]:
```suggestion
  depends_on xcode: ["12.5", :build]
```
This needs Swift 5.4, which comes with Xcode 12.5.

Comment (cho-m) [reply Formula/publish.rb:17]: Do we have a workaround for Catalina build yet?

Comment (carlocab) [reply Formula/publish.rb:17]: Nope, no workaround for a Catalina build. Upstream also explicitly requires Big Sur, but we might be able to do a Catalina build with Swift 5.4.

**Review by carlocab — APPROVED** [review]
I think this is good to merge now. We can bottle it for Catalina later when we have Swift 5.4.

**Comment by chenrui333** [comment]
agree, i think we should ship the upgrade and then we can rebuild the formula when we have swift 5.4

**Pull request closed without merging**